src/main/java/automaticlotto/domain/LottoRanking.java

```diff
@@ -0,0 +1,31 @@
+package automaticlotto.domain;
+
+import java.util.stream.Stream;
+
+public enum LottoRanking {
+
+    MISMATCH(0, 0),
+    THREE(3, 5000),
```

Comment:
```suggestion
    THREE(3, 5_000),
```
언더바를 통해 가독성을 높일 수 있습니다 ^_^